website/src/frontend-scripts/contributors.ts

```diff
@@ -0,0 +1,60 @@
+const applyStyles = (element: any, styles: any) => {
```

Comment:
Seems this code is not properly formatted, you can format it using `pnpm format`